### DISCUSSION TITLE
Dynamic cookie domain

### DISCUSSION THREAD

**Discussion by chladog** [payloadcms/payload#662] [Feature Requests & Ideas]
In similar fashion to #661 - it would be great to be able to set the domain parameter of cookie dynamically based on the context/request. E.g. my app is running on multiple domains and I want to set domain parameter based on request origin to allow cookie sharing across subdomains.
Currently only static config is possible.

_It would be possible to allow for dynamic config for all cookie options: ```secure, sameSite, expires, path``` as needs can vary based on origin._

**Comment by jmikrut** [comment]
@chladog this is a great idea. 

We'd accept a PR to do this for sure. Otherwise, we can get to it but we have a handful of initiatives we're working through now that will take precedence. If you decide to tackle this, I'd be up for strategizing!
